spk/sickchill/src/service-setup.sh

```diff
@@ -24,12 +24,17 @@ service_postinst ()
         ${GIT} clone --depth 10 --recursive -q -b master git://github.com/SickChill/SickChill.git ${SYNOPKG_PKGDEST}/var/SickChill > /dev/null 2>&1
 
       if [ -n "${wizard_username}" ] && [ -n "${wizard_password}" ]; then
-        mkdir -p ${SC_DATA_DIR}
-        cat << EOF > ${SC_CFG_FILE}
+        if [ -f "${SC_CFG_FILE}" ]; then
+          sed -i "/^\s*web_username\s*=/s/\s*=\s*.*/ = ${wizard_username}/" ${SC_CFG_FILE}
```

Comment:
This could be risky if, in the future, there is a new setting called "web_username" outside "General" section, but as you are the owner, I suppose you will ensure this never happens

Reply:
There shouldn't be anything outside general section

Reply:
There is an extra line in there I couldn't remove on my phone smh